### ISSUE TITLE
Automatic program not working anymore :(

### ISSUE THREAD

**Issue by makitstop** [milkey-mouse/SteamBulkSeller#1]
i'm pretty bad with technology, so i've been trying to use the automatic link thing, but i think the code is broken on newer versions of steam, all it did was say that i'm not on my steam inventory, even when i am, and break the webpage, any troubleshooting you can do if you're still active?

**Comment by makitstop** [comment]
Update, turns out, because i had to reset my password to log into steam on my browser, i can't sell anything, so that might explain my issues, i'll keep this up just in case, but i'll update ya'll when i'm able to sell stuff again

**Comment by makitstop** [comment]
Update 2: yeah it's still not working, the same exact thing is happening but now instead of breaking the webpage, it tries to sell the gems in my inventory and then breaks right after

**Comment by twilligon** [comment]
Yeah sorry if this doesn't work anymore, I'd be surprised if Steam hasn't made a breaking change in the last decade or so. These days I wouldn't necessarily rely on code I wrote when I was 13 :P
